Libplanet/Net/Transports/NetMQTransport.cs

```diff
@@ -44,6 +44,7 @@ public class NetMQTransport : ITransport
 
         private Channel<MessageRequest> _requests;
         private long _requestCount;
+        private long _socketCount;
```

Comment:
How about grouping fields used only for logging, and leaving a comment for these fields?

```suggestion
        // Used only for logging:
        private long _requestCount;
        private long _socketCount;

        private Channel<MessageRequest> _requests;
```

Reply:
Fixed. See the updated file changes.